openssl/src/ssl/mod.rs

```diff
@@ -2456,6 +2498,33 @@ impl SslRef {
         }
     }
 
+    /// Derives keying material for application use in accordance to RFC 5705.
+    ///
+    /// Requires OpenSSL 1.1.1 or newer.
+    ///
+    /// This corresponds to [`SSL_export_keying_material_early`].
+    ///
+    /// [`SSL_export_keying_material_early`]: https://www.openssl.org/docs/manmaster/man3/SSL_export_keying_material_early.html
+    #[cfg(ossl111)]
+    pub fn export_keying_material_early(
+        &self,
+        out: &mut [u8],
+        label: &str,
+        context: &[u8],
```

Comment:
I believe we'll want the context argument to be optional like it is in export_keying_material.

Reply:
This function is only usable with TLS1.3, where there is no distinction between an absent and an empty context. The C API is not capable of expressing that distinction either, for the same reason.

Reply:
Ah right. Might be worth adding a note to the docs here explaining why the API differs from export_keying_material?